### ISSUE TITLE
[Cemu] Invalid account.dat

### ISSUE THREAD

**Issue by AToska21** [MatthewL246/pretendo-docker#139]
Hi! It's me again, lol.
When I generate Cemu online files through the website or the script, Cemu overwrites it with a blank account.dat file with default values.
I have no idea how to fix this, nor is there any helpful log I could find.

**Comment by AToska21** [comment]
Update: the website's generator works fine. It just makes an invalid password hash so I used a tool to remake it and it works fine. I will have to look into the script further.

**Comment by MatthewL246** [comment]
The website and the script should be generating very similar account.dat files since I used some of the code from the website in my script.

Was Cemu open while you were copying the new account.dat? Maybe it only loads changes in the file at startup.

**Comment by AToska21** [comment]
> The website and the script should be generating very similar account.dat files since I used some of the code from the website in my script.
> 
> Was Cemu open while you were copying the new account.dat? Maybe it only loads changes in the file at startup.

No, always restarted Cemu. Very odd since from the website it always worked.

**Comment by AToska21** [comment]
Still not sure what the bug is, but I figured out the issue with the website not generating the correct password hash. The site is supposed to ask the user for their password when downloading the online files, but it doesn't.

**Comment by MatthewL246** [comment]
Strange, it does appear to be working for me when I click the download button:

![image](https://github.com/user-attachments/assets/5d943872-a43c-4a32-9cd3-740944924894)

I would prefer to move to the script approach, though, because it is possible that future website updates will break the patch that restores this feature, and it's much easier to maintain a script than a large patch.
